apps/store/src/app/[locale]/checkout/CheckoutPage.tsx

```diff
@@ -129,16 +137,50 @@ export function CheckoutPage({ locale }: { locale: RoutingLocale }) {
             </Heading>
           </header>
 
-          <BankIdContextProvider>
-            <CheckoutForm
-              shopSessionId={shopSession.id}
-              ssn={shopSession.customer.ssn}
-              cart={shopSession.cart}
-              shouldCollectName={getShouldCollectName(shopSession.customer)}
-              shouldCollectEmail={getShouldCollectEmail(shopSession.customer)}
-            />
-            <BankIdDialog />
-          </BankIdContextProvider>
+          <Card.Root variant="secondary">
+            <Text>{t('checkout:ORDER_SUMMARY_TITLE')}</Text>
+
+            <OrderBreakdown cart={shopSession.cart} />
+
+            <Divider />
+
+            {shouldShowDiscountSection ? (
+              <>
+                <CartDiscount shopSession={shopSession} />
+                <Divider />
+              </>
+            ) : null}
```

Comment:
Could this simplified `&&`?
```suggestion
            {shouldShowDiscountSection && (
              <>
                <CartDiscount shopSession={shopSession} />
                <Divider />
              </>
            ) }
```

Reply:
I prefer ternaries for conditional rendering in JSX since they avoid the pitfalls of the `&&` operator. The rational is better explained in this video.

https://www.youtube.com/watch?v=Rr5AqASIyxw

Reply:
Let's have a discussion about conditional rendering in the FE forum tomorrow.